custom-connectors/secplugs/apiDefinition.swagger.json

```diff
@@ -0,0 +1,445 @@
+                },
+                "meta_data": {
+                  "type": "object",
+                  "properties": {
+                    "plugin_info": {
+                      "type": "object",
+                      "properties": {
+                        "type": {
+                          "type": "string",
+                          "description": "type"
+                        },
+                        "name": {
+                          "type": "string",
+                          "description": "name"
+                        }
+                      },
+                      "description": "plugin_info"
+                    },
+                    "capability": {
+                      "type": "string",
+                      "description": "capability"
+                    },
+                    "vendor_info": {
+                      "type": "object",
+                      "properties": {
+                        "vendor_config_name": {
+                          "type": "string",
+                          "description": "vendor_config_name"
+                        },
+                        "entitlement_level": {
+                          "type": "string",
+                          "description": "entitlement_level"
+                        },
+                        "vendor": {
+                          "type": "string",
+                          "description": "vendor"
+                        },
+                        "credits": {
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "credits"
+                        },
+                        "params": {
+                          "type": "object",
+                          "properties": {},
+                          "description": "params"
+                        }
+                      },
+                      "description": "vendor_info"
+                    }
+                  },
+                  "description": "meta_data"
+                },
+                "report_id": {
+                  "type": "string",
+                  "description": "report_id"
+                },
+                "verdict": {
+                  "type": "string",
+                  "description": "verdict"
+                },
+                "duration": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "duration"
+                },
+                "user_report_url": {
+                  "type": "string",
+                  "description": "user_report_url"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Scan files using SecPlug Scan",
+        "operationId": "filescan",
+        "parameters": [
+          {
+            "in": "header",
+            "name": "x-api-key",
+            "type": "string",
+            "default": "GW5sb8sj8D9CtvVrjsTC22FNljxhoVuL1UoM6fFL",
```

Comment:
@Ajaatsecplugs It looks like a key or secret which are not allowed to mention in any connector artifacts due to security concerns.

Reply:
this is an open default key, available for public.

Reply:
@Ajaatsecplugs Is it possible to remove it form the code